.gitignore

```diff
@@ -3,6 +3,8 @@
 /packaging/windows/build/
 /packaging/linux/build/
 /bindings/build*/
+/example/dev/build*/
```

Comment:
we should add to gitignore all the build directories of the examples but personally I didn't find a smart way to do it.

Reply:
IMHO the best would be to integrate example compilation in the main CMakeLists, so that examples will go in the main build folder along with all other stuff.